smos/src/Smos/Actions/Entry.hs

```diff
@@ -100,10 +101,20 @@ entrySelectTags =
     { actionName = "entrySelectTags",
```

Comment:
Let's rename this to `entrySelectTagsFromStart` to be nicely symmetrical with `entrySelectTagsFromBack`.
",

Reply:
I was thinking about this as well, but I wasn't so sure that is it a good idea to rename existing action